src/metabase/perf_util.clj

```diff
@@ -0,0 +1,71 @@
+(ns metabase.perf-util
```

Comment:
What do you think about renaming this something like `metabase.util.performance.iter-tools`?

There are two reasons:

1. This avoids introducing  new module (we use condo to restrict the metabase.X.* to metabase.Y.* requires)
2. This makes the nature of these two functions clearer from the namespacing where they're used.

Also, these seem like very obvious functions, is there not a library somewhere with these and more?

Reply:
I'll put them wherever you tell me:). I have no preference here.

> Also, these seem like very obvious functions, is there not a library somewhere with these and more?

Perhaps there is, but I wouldn't pull a whole lib just for these two functions. Besides, this gives us the flexibility to tweak something if needed. For example, add more arities.

Reply:
I'm happy even with `metabase.util.performance`, but yeah let's avoid creating a new module then.